### ISSUE TITLE
Investigate markdown macro option

### ISSUE THREAD

**Issue by VTimofeenko** [SabbathHex/confluence_poster#35]
Confluence has some form of markdown macro that converts markdown text in online editor into confluence. Maybe it is possible to emulate this behavior on command line to allow writing articles in markdown and convert them when posting or prepend a macro if a flag is supplied

Credit to @iklokov for idea

**Comment by VTimofeenko** [comment]
Seems to be possible according to the answer [here](https://community.atlassian.com/t5/Answers-Developer-Questions/How-do-you-post-markdown-using-confluences-rest-API/qaq-p/492056).

Works on server version:
```
> curl -XPOST -u 'admin:pwd' http://localhost:8090/rest/tinymce/1/markdownxhtmlconverter -H
'Content-Type: application/json' -d '{"wiki": " * one\n * two"}'
<ul> <li>one <li>two </ul> %
> 
```

**Comment by VTimofeenko** [comment]
- [x] Add page file format to page model
- [ ] Add source page file format to post-page:
    - [x] confluencewiki
    - [x] markdown
    - [x] editor (aka html)
    - [ ] return error(?) if "storage", "export_view" "view"
- [x] If file format was not specified - try to guess the file format during runtime
- [x] Check how well the approach with markdown as embedded storage works
- [x] Add command to convert markdown using the Confluence instance
    - [x] Command shows a warning about using undocumented method and suggest using other tools
    - [x] Command using 'editor' representation
- [x] Add format to config wizard as optional
- [x] Add a section about format to README

**Comment by VTimofeenko** [comment]
>Check how well the approach with markdown as embedded storage works

On a blank Confluence instance (without any custom-installed macros) this approach:

```
<ac:structured-macro ac:name="markdown"><ac:plain-text-body><![CDATA[
*Markdown goes here*
]]></ac:plain-text-body></ac:structured-macro>
```

does not work

**Comment by VTimofeenko** [comment]
Released as part of #38
